pkg/workloads/runtimes.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workloads
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+func init() {
+	containerRuntimes = make(map[containerRuntimeType]containerRuntimeOpts)
+	runtimeDefaultsOpts = map[containerRuntimeType]containerRuntimeOpts{
+		ContainerD: {
+			Endpoint: "unix:///var/run/docker.sock",
+		},
+	}
+
+}
+
+const (
+	None       containerRuntimeType = "none"
+	Auto       containerRuntimeType = "auto"
+	ContainerD containerRuntimeType = "docker"
```

Comment:
The rename of the package to `containerd` was a bit premature as it is still docker specific. Let's continue calling the variable Docker so we can name the containerd variable properly when we add support for it.

Reply:
Done